test/regress/regress3/bv_to_int_check_bvsgt_bvlshr0_4bit.smt2.minimized.smt2

```diff
@@ -1,6 +1,7 @@
 ; COMMAND-LINE:  --solve-bv-as-int=bv 
 ; COMMAND-LINE:  --solve-bv-as-int=sum     
 ; COMMAND-LINE:  --solve-bv-as-int=iand --iand-mode=sum    
+; COMMAND-LINE:  --solve-bv-as-int=iand --iand-mode=bitwise
```

Comment:
I would also suggest adding this to:
https://github.com/CVC4/CVC4/blob/master/test/regress/regress1/nl/iand-native-1.smt2
https://github.com/CVC4/CVC4/blob/master/test/regress/regress1/nl/iand-native-2.smt2

Reply:
Thanks, added with 7ca81d943baa8118665670245fda4368461d66b1